ui-tests/src/test/resources/features/wip/data-buckets.feature

```diff
@@ -161,3 +162,215 @@ Feature: Test functionality of data buckets during creation of integration
+
+
+
+    ##################### step NEW step ##################################
+    And She adds integration "connection" on position "0"
+    When Camilla selects the "PostgresDB" connection
+    And she selects "Invoke SQL" integration action
+    Then she fills invoke query input with "select first_name as myName from contact where company = :#COMPANY and last_name = :#MYSURNAME limit 1;" value
+    And clicks on the "Done" button
+
+
+    ##################### step NEW step step ##################################
+
+    And She adds integration "connection" on position "0"
+    When Camilla selects the "PostgresDB" connection
+    And she selects "Invoke SQL" integration action
+    Then she fills invoke query input with "select last_name as mySurname from contact where company = :#COMPANY limit 1;" value
+    And clicks on the "Done" button
+
+    ##################### step step step NEW step ##################################
+
+    And She adds integration "connection" on position "2"
+    When Camilla selects the "PostgresDB" connection
+    And she selects "Invoke SQL" integration action
+    Then she fills invoke query input with "select lead_source as myLeadSource from contact where company = :#COMPANY and first_name = :#MYNAME and last_name = :#MYSURNAME limit 1;" value
+    And clicks on the "Done" button
+
+    ##################### step step step step NEW step ##################################
+
+    And She adds integration "connection" on position "3"
+    When Camilla selects the "PostgresDB" connection
+    And she selects "Invoke SQL" integration action
+    Then she fills invoke query input with "insert into contact values (:#COMPANY , :#MYNAME , :#MYSURNAME , :#LEAD, '1999-01-01')" value
+    And clicks on the "Done" button
+
+    ##################### check warnings ##################################
+
+    And she checks that there is no warning inside of step number "0"
+
+    Then she checks that text "Add a data mapping step" is "visible" in step warning inside of step number "2"
+    Then she checks that text "Add a data mapping step" is "visible" in step warning inside of step number "4"
+    Then she checks that text "Add a data mapping step" is "visible" in step warning inside of step number "6"
+    Then she checks that text "Add a data mapping step" is "visible" in step warning inside of step number "8"
+    Then she checks that text "Add a data mapping step" is "visible" in step warning inside of step number "10"
+
+
+
+
+    ##################### mappings ##################################
+    #################################################################
+
+    ##################### step step step step step MAPPING step ##################################
+
+    And She adds integration "step" on position "4"
+    And she selects "Data Mapper" integration step
+    Then she is presented with data mapper ui
+
+    Then check that data bucket "1 - SQL Result" is available and opens it
```

Comment:
`Then check data bucket "xy" is available and open it` without a bit of grammar polish :)
 
But I'm thinking about a different approach like
```
When check data bucket "xy" is available
Then open data bucket "xy"
```
or 
```
Then check data bucket(s) and perform action(s)
  | Data Bucker name | state | action |
```

@mcada Wdyt?

Reply:
Data buckets are closed by default so when creating integration everything you need is this single step I created. Since this feature is testing if data bucket exists, no other feature will need just open/check - no point to test one thing more times. Close data bucket step may be useful, I am going to create it.

Reply:
Ok, I'd like to tune down the repetitive step definition, that was the motication of step's table proposal(last one).